docs/components/fusee.md

```diff
@@ -0,0 +1,44 @@
+
+Fusée is split into two separate parts, fusée-primary and fusée-secondary. This is due to the RCM Vulnerability only allowing payloads of a limited filesize to be sent to the device.
+
+As of June 2018, there are new Switch systems being sold that prevent Fusée (or any payload that requires the Fusée Gelée exploit) from working due to having an ipatched bootrom. All ipatched systems share the HAC-S-JXE-C3 product code. While Fusée cannot work on these ipatched units, they still come on firmware 4.1.0, which is vulnerable to the upcoming Déja Vu software exploit. Note that if you update past 4.1.0 on one of these ipatched units your odds of being able to install Atmosphère or run any homebrew become practically non-existent.
+
+Additionally, a hardware revision of the Switch known as “Mariko” is believed to be in development. No such units have been seen in stores yet, but it is expected Nintendo will roll them out silently. The Mariko units will most likely patch the bootrom vulnerability Fusée Gelée, which is currently used to access CFW, and will likely have their own proprietary bootloader.
+
+## Fusée-Primary
+Fusée-primary is the payload file (fusee-primary.bin) sent to the Switch from an external device. Once sent, fusée-primary makes initial preparations before loading fusée-secondary from the Switch’s SD Card.
+
+Fusée-primary can be configured via the BCT.ini file located on the Switch’s SD card.
+
+## Fusée-Secondary
+Fusée-secondary is a payload file that stays on the root of the Switch’s SD Card (fusee-secondary.bin). It is automatically launched once fusée-primary has finished, and is responsible to preparing the Switch’s hardware for future running environments, such as the homebrew menu. Fusée-secondary also is responsible for validating and launching Exosphère.
+
+Fusée-secondary contains various .kip [modules](https://github.com/joelc608/Atmosphere/tree/master/docs/modules). These modules modify existing features in the OS, and can also add new ones.
+
+Fusée is also capable of chainloading other environments including Linux and many more.
+
+## BCT.ini
+BCT.ini is the configuration file used by fusée-primary and fusée-secondary. It is read by fusee-primary.bin to setup and boot fusee-secondary.bin and is also read by fusee-secondary.bin to configure Exosphère or to specify the environment it should boot.
+
+### Configuration
+This file is located at the root of your SD.
+```
+BCT0
+[stage1]
+stage2_path = fusee-secondary.bin
+stage2_addr = 0xF0000000
+stage2_entrypoint = 0xF0000000
+```
+Add the following lines and replace the `X` according to the following list if you have trouble booting past the FW version detection.
+`target_firmware` is the OFW major version. 
+```
+[exosphere]
+target_firmware = X
+```
+```
+1.0.0 1
+2.X.X 2
```

Comment:
Is this table incomplete?

Reply:
Wasn't meant to be a table. The 1.0 and 2.0 exist to serve as examples of how to format the code for each firmware. If you want us to make it a table we can easily do it though.